### PR TITLE
Add Ω to latex_symbols

### LINKED ISSUE #31589: Add ohm `Ω` to latex symbols
It would be nice if the symbol for 'OHM SIGN' (U+2126), i.e. Ω were added to [this file](https://github.com/JuliaLang/julia/blob/5a4843ec242fb0c5655fba97b38ea502eed6861d/stdlib/REPL/src/latex_symbols.jl). It's commonly used in electrical engineering for the unit of resistance. The symbol for 'INVERTED OHM SIGN' (U+02127), i.e. ℧ is already in the file and can be accessed using `\mho`, so it'd be best if this were accessible using `\ohm`.

### PR DESCRIPTION
**Pull request by kdheepak**
Resolves #31589

### CODE REVIEW

File: stdlib/REPL/src/latex_symbols.jl [hunk 1]
```diff
@@ -436,6 +436,7 @@ const latex_symbols = Dict(
     "\\Re" => "ℜ",
     "\\xrat" => "℞",
     "\\trademark" => "™",
+    "\\ohm" => "Ω",
```
Comment (stevengj) [inline stdlib/REPL/src/latex_symbols.jl]: `Ω` here is U+03a9, but it should be U+2126?

**Comment by stevengj** [comment]
It should be added in the `# manual additions:` section at the top of the file, just so that we retain some sense of when/how the symbols were added.

File: stdlib/REPL/src/latex_symbols.jl (hunk 1 above)
Comment (kdheepak) [reply stdlib/REPL/src/latex_symbols.jl]: Oh interesting, I thought they normalized to the same utf-8 character in the file. How can you tell it is U+03a9?

Comment (stevengj) [reply stdlib/REPL/src/latex_symbols.jl]: The normalization doesn't happen in the file, it happens only when Julia parses it.  To see what codepoint it was, I just copied and pasted it into the REPL from this PR:
```jl
julia> 'Ω'
'Ω': Unicode U+03a9 (category Lu: Letter, uppercase)
```

File: stdlib/REPL/src/latex_symbols.jl [hunk 2]
```diff
@@ -111,6 +111,7 @@ const latex_symbols = Dict(
     "\\impliedby" => "⟸",
     "\\to" => "→",
     "\\euler" => "ℯ",
+    "\\ohm" => "\u2126",
```
Comment (stevengj) [inline stdlib/REPL/src/latex_symbols.jl]: Use `"Ω"` here to match the style of the other code.

File: stdlib/REPL/src/latex_symbols.jl (hunk 1 above)
Comment (kdheepak) [reply stdlib/REPL/src/latex_symbols.jl]: Neat! I didn't know I could copy paste it and display it as a character in the Julia REPL.

File: stdlib/REPL/src/latex_symbols.jl (hunk 2 above)
Comment (kdheepak) [reply stdlib/REPL/src/latex_symbols.jl]: Resolved in 766bc0a.

**Review by stevengj — APPROVED** [review]

**Pull request merged**